go-controller/pkg/ovn/address_set/address_set.go

```diff
@@ -437,6 +441,31 @@ func (as *ovnAddressSets) AddIPs(ips []net.IP) error {
 	return nil
 }
 
+// TEMP
+func (as *ovnAddressSets) AddIPsReturnsOps(ips []net.IP) ([]ovsdb.Operation, error) {
+	var ops []ovsdb.Operation
+	var err error
+	if len(ips) == 0 {
+		return nil, nil
+	}
+
+	v4ips, v6ips := splitIPsByFamily(ips)
+	if as.ipv6 != nil {
+		if ops, err = as.ipv6.addIPsReturnsOps(v6ips); err != nil {
```

Comment:
@tssurya this will only work for single-stack, I think you want to `ops = append(ops, <something>)`?

Reply:
yes I did (facepalm at my carelessness)